crates/uv/src/commands/reporters.rs

```diff
@@ -201,6 +266,9 @@ impl uv_installer::DownloadReporter for DownloadReporter {
     }
 
     fn on_complete(&self) {
+        // Need an extra call to `set_message` here to fully clear avoid leaving ghost output
+        // in Jupyter notebooks.
+        self.reporter.root.set_message("");
```

Comment:
This bug existed before the new progress bars, where if the previous progress bar line was longer than the next one, it wouldn't fully clear. Seems like a bug in indicatif.